includes/wikia/services/templates/Pagination_index.php

```diff
@@ -1,26 +1,26 @@
-<ul class="Pagination"  <? foreach($data as $key => $val ): ?> data-<?php echo $key ?>="<?php echo htmlentities($val); ?>" <? endforeach; ?> >
+<ul class="Pagination"  <? foreach($data as $key => $val ): ?> data-<?= htmlspecialchars( $key ) ?>="<?php echo htmlentities($val); ?>" <? endforeach; ?> >
 	<? if( $prev ): ?>
 		<li data-page="<?= $currentPage - 1; ?>"  class="prev">
-			<a href="<?= ($url !== '#' ? $url.($currentPage - 1) : $url) ?>"><?= $prevMsg ?></a>
+			<a href="<?= htmlspecialchars( $url !== '#' ? $url.($currentPage - 1) : $url) ?>"><?= htmlspecialchars( $prevMsg ) ?></a>
```

Comment:
For attributes `Sanitizer::encodeAttribute` is preferred over plain `htmlspecialchars` (both are safe in this instance though due to the double quotes).